lib/sles4sap/sap_deployment_automation_framework/naming_conventions.pm

```diff
@@ -370,3 +372,62 @@ sub get_workload_vnet_code {
     # if it is too long you might hit name length limit and test ID gets clipped.
     return ($args{job_id});
 }
+
+=head2 get_sdaf_inventory_path
+
+    get_sdaf_inventory_path();
+
+Returns full Ansible inventory filepath respective to deployment type.
+
+=over
+
+=item * B<env_code>:  SDAF parameter for environment code (for our purpose we can use 'LAB')
+
+=item * B<sdaf_region_code>: SDAF parameter to choose PC region. Note SDAF is using internal abbreviations (SECE = swedencentral)
+
+=item * B<vnet_code>: SDAF parameter for virtual network code. Library and deployer use different vnet than SUT env
+
+=item * B<sap_sid>: SDAF parameter for sap system ID.
+
+=back
+=cut
+
+sub get_sdaf_inventory_path {
+    my (%args) = @_;
+
+    # Argument (%args) validation is done by 'get_sdaf_config_path()'
+    my $config_root_path = get_sdaf_config_path(
+        deployment_type => 'sap_system',
+        sap_sid => $args{sap_sid},
+        env_code => $args{env_code},
+        vnet_code => $args{vnet_code},
+        sdaf_region_code => $args{sdaf_region_code}
+    );
+
+    # file name is hard coded in SDAF
+    return "$config_root_path/$args{sap_sid}_hosts.yaml";
+}
+
+=head2 deployer_private_key_path
+
+    deployer_private_key_path();
+
+Returns path leading to deployer VM private key. Located on worker VM.
+
+=cut
+
+sub deployer_private_key_path {
+    return ('~/.ssh/id_rsa');
+}
```

Comment:
Isn't a function a little overkill for this? It could be a simple package variable, for example:
```
our @EXPORT = qw(
...
    $deployer_private_key_path
);
...
our $deployer_private_key_path = '~/.ssh/id_rsa';
```
Or even a constant if we don't expect this to change.

Reply:
Hmm... I have forgotten about package variables being an option. 
I do expect this to get a bit more complicated further down the road... but for now I will change it to package variables.   
Thanks for the tip.